src/clue/app-config.json

```diff
@@ -12,6 +12,7 @@
   "defaultLearningLogTitle": "UntitledLog",
   "initialLearningLogTitle": "My First Learning Log",
   "defaultLearningLogDocument": true,
+  "documentLabelProperties": [],
```

Comment:
You shouldn't need this line. In the model, `types.array(types.string)` will default to an empty array and for configuration purposes clients should be able to ignore/not specify properties they don't care about.

Reply:
I thought it'd be good to include it as an example but that makes sense, removed from `app-config`!